cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server_test.go

```diff
@@ -38,7 +46,10 @@ import (
 	"k8s.io/client-go/kubernetes"
 	typfake "k8s.io/client-go/kubernetes/fake"
 	k8stesting "k8s.io/client-go/testing"
-	"k8s.io/helm/pkg/proto/hapi/chart"
+
+	// TODO(mnelson): models.Chart.Maintainers is depending on the old v1 chart
+	// code. I don't expect there is any reason other than a historical one.
+	chartv1 "k8s.io/helm/pkg/proto/hapi/chart"
```

Comment:
But, I guess it will be removed in Helm 3.6.2+ versions (though we are unable to upgrade due to the k8s 1.22 upgrade, no?)

Reply:
We could probably update `models.Chart.Maintainers` now, regardless of the later helm version. Just didn't want to do that here.